.drone.yml

```diff
@@ -126,6 +127,27 @@ pipeline:
     commands:
       - cd /var/www/owncloud/server/
       - bash ./apps/user_ldap/tests/acceptance/setConfig.sh
+    when:
+      matrix:
+        NEED_SERVER: true
+
+  ldaps-config-for-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/server/
+      - php occ ldap:set-config LDAPTestId ldapPort "636"
+      - php occ ldap:set-config LDAPTestId ldapHost "ldaps://ldap"
```

Comment:
Setting `ldaps://` doesnot require us to set `ldapTLS`?

Reply:
actually both cannot be set at the same time. I had to remove `ldapTLS=1` again